src/lib/form-field/form-field.ts

```diff
@@ -229,18 +229,18 @@ export class MatFormField extends _MatFormFieldMixinBase
    * @deprecated
    * @breaking-change 7.0.0
    */
-  @ViewChild('underline') underlineRef: ElementRef;
-
-  @ViewChild('connectionContainer') _connectionContainerRef: ElementRef;
-  @ViewChild('inputContainer') _inputContainerRef: ElementRef;
-  @ViewChild('label') private _label: ElementRef;
-  @ContentChild(MatFormFieldControl) _control: MatFormFieldControl<any>;
-  @ContentChild(MatPlaceholder) _placeholderChild: MatPlaceholder;
-  @ContentChild(MatLabel) _labelChild: MatLabel;
-  @ContentChildren(MatError) _errorChildren: QueryList<MatError>;
-  @ContentChildren(MatHint) _hintChildren: QueryList<MatHint>;
-  @ContentChildren(MatPrefix) _prefixChildren: QueryList<MatPrefix>;
-  @ContentChildren(MatSuffix) _suffixChildren: QueryList<MatSuffix>;
+  @ViewChild('underline') underlineRef!: ElementRef;
+
+  @ViewChild('connectionContainer') _connectionContainerRef!: ElementRef;
+  @ViewChild('inputContainer') _inputContainerRef!: ElementRef;
+  @ViewChild('label') private _label!: ElementRef;
+  @ContentChild(MatFormFieldControl) _control!: MatFormFieldControl<any>;
+  @ContentChild(MatPlaceholder) _placeholderChild!: MatPlaceholder;
+  @ContentChild(MatLabel) _labelChild!: MatLabel;
+  @ContentChildren(MatError) _errorChildren!: QueryList<MatError>;
+  @ContentChildren(MatHint) _hintChildren!: QueryList<MatHint>;
+  @ContentChildren(MatPrefix) _prefixChildren!: QueryList<MatPrefix>;
+  @ContentChildren(MatSuffix) _suffixChildren!: QueryList<MatSuffix>;
```

Comment:
Optional for prefix, suffix, error, hint, and anything else that might not actually be there?

Reply:
We can do it for the ones that are a `ContentChild`, but the `ContentChildren` are guaranteed to be a `QueryList`, even if they didn't match anything.